### DISCUSSION TITLE
How do you backup your secret?

### DISCUSSION THREAD

**Discussion by jackfang888** [Authenticator-Extension/Authenticator#778] [Q&A] [answered]
Given the [problem with Vivaldi](https://forum.vivaldi.net/topic/67228/extension-authenticator-crashes/20) I want to transfer my secret to another browser, however I cannot find documentation on how to do this.  I may be dumb and missed it, but I've looked in:

https://github.com/Authenticator-Extension/Authenticator/discussions

https://authenticator.cc/docs/en/

https://www.google.com/search?q=backup+%22authenticator+extension%22+secret+-%22google+authenticator%22

![image](https://user-images.githubusercontent.com/20893390/138757218-0ad759cb-5103-47f6-bc06-4334acd403ba.png)

Yeah, *but how?*

**Comment by mymindstorm** [comment, marked as answer]
Open the extension -> Menu -> Backup -> Download password-protected backup file

**Reply by jackfang888** [reply]
Just to be clear, when you say "open the extension" you mean single left click the icon?

![image](https://user-images.githubusercontent.com/20893390/138929121-38d070cc-f0ef-461a-9a6e-5d5c99c6604c.png)

**Reply by mymindstorm** [reply]
Yes

**Reply by jackfang888** [reply]
Well due to it being broken in Vivaldi, if I click the extension button it immediately crashes; there are no options so it is currently impossible:

![image](https://user-images.githubusercontent.com/20893390/138951347-06096a87-9ee1-42e2-8325-1f5226a15d09.png)


Is there another way to create a backup?
